src/pages/generate/pages/loading/ProgressBar.tsx

```
@@ -8,9 +8,18 @@ interface ProgressLoadingBarProps {
   onComplete?: () => void;
 }
 
+// 로테이션할 안내 문구 배열
+const LOADING_MESSAGES = [
+  '이미지를 생성하는 중이에요',
```

ESLint

[error] Replace `'이미지를·생성하는·중이에요'` with `"이미지를·생성하는·중이에요"`

(prettier/prettier)